app/views/activities/index.html.slim

```diff
@@ -1,5 +1,5 @@
 div.row
-  div.col-sm-12
+  div.col-sm-12 id="activities-body"
```

Comment:
can you change this to `div.col-sm-12#activities-body`, it's more compact and easy to read